spinalcordtoolbox/scripts/sct_download_data.py

```diff
@@ -42,8 +42,7 @@
     },
     "PAM50": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/PAM50/releases/download/r20201104/PAM50-r20201104.zip",
-            "https://osf.io/download/5fa21326a5bb9d00610a5a21/",
+            "https://github.com/spinalcordtoolbox/PAM50/releases/download/r20220527/PAM50-r20220527.zip",
```

Comment:
The one downside to adding the histology microstructure atlas to the PAM50 template is that the installation size increases quite a bit:

* Before: [r20201104](https://github.com/spinalcordtoolbox/PAM50/releases/tag/r20201104) (67.6 MB)
* After: [r20220527](https://github.com/spinalcordtoolbox/PAM50/releases/tag/r20220527) (156 MB)

Is this an acceptable increase given the usefulness of the histology microstructure atlas?

Reply:
(from internal discussion): Might not need to be included in the mainline PAM50 template – could be a separate sct_download_data download.